apps/web/src/lang/fr.json

```diff
@@ -6806,12 +6782,28 @@
     "defaultMessage": "Nom du gestionnaire",
     "description": "Label for the user select field on team membership form"
   },
-  "hr/i9h": {
-    "defaultMessage": "Sélectionnez une équipe...",
-    "description": "Placeholder displayed for team selection input."
+  "Vf+lsG": {
+    "defaultMessage": "Nom de l’équipe (Français)",
+    "description": "Label for the French team display name input"
+  },
+  "XNBegf": {
+    "defaultMessage": "Brève description de l’équipe (Français)",
+    "description": "Label for the French team description input"
+  },
+  "YjMfWZ": {
+    "defaultMessage": "Nom de l’équipe (Français)",
+    "description": "Name of an organization/team in French field."
   },
-  "mOS8rj": {
-    "defaultMessage": "Équipe principale",
-    "description": "Label displayed for selecting what team a new pool belongs to."
```

Comment:
this too

Reply:
Added back in [add lost translations back](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5921/commits/657b8652fd1d68848c65a3f5c5d5a879b4a536f4)